### ISSUE TITLE
Travis CI build failing because get_category_link is prohibited

### ISSUE THREAD

**Issue by olefredrik** [WordPress-Coding-Standards/WordPress-Coding-Standards#493]
Hi guys. I just noticed any new pull requests on my project will [fail the Travis CI build](https://travis-ci.org/olefredrik/FoundationPress/jobs/91794090#L235) because of this error

```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 137 | ERROR | get_category_link is prohibited, please use
     |       | wpcom_vip_get_term_link() instead.
     |       | (WordPress.VIP.RestrictedFunctions.get_term_link)
----------------------------------------------------------------------
```

This is new to me. Isn't get_category_link allowed anymore? If so, why is that? What can I do to make my Travis CI builds pass? I'll appreciate any help on this.

**Comment by westonruter** [comment]
@olefredrik VIP recently updated their list of [Uncached Functions](https://vip.wordpress.com/documentation/caching/uncached-functions/). 

They now require using `wpcom_vip_get_term_link()` instead of `get_category_link()`.

See #491

**Comment by olefredrik** [comment]
@westonruter : allright, thanks! I guess that it also applies even if I'm not enrolled as a VIP member?

**Comment by westonruter** [comment]
@olefredrik ah, no. So you should be excluding this sniff entirely then, since these are specific to WordPress.com VIP.

**Comment by olefredrik** [comment]
@westonruter : Ok, thanks for the clarification. Last question: Do you know the correct codesniffer.ruleset.xml syntax for excluding this sniff? Is there a list somewhere of all sniffs that run by default?

Appreciate your feedback!

**Comment by JDGrimes** [comment]
@olefredrik : you would add something like this to your ruleset.xml file to exclude that sniff:

```xml
<rule ref="WordPress-VIP">
		<exclude name="WordPress.VIP.RestrictedFunctions" />
</rule>	
```
